stylesheets/stylePost.ts

```diff
@@ -0,0 +1,22 @@
+import { StyleSheet } from "react-native";
```

Comment:
komplettes Stylesheet hier unnötig. Siehe Kommentare oben. Alles direkt mit Tailwind in den Code. Stylesheet danach wieder löschen

Reply:
bitte noch löschen